unity/Assets/0Refactored/scenes/experiments/HuygensPrinciple/scripts/InterferencePlate.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Maroon.Physics;
+using Maroon.Physics.HuygensPrinciple;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InterferencePlate : MonoBehaviour
+{
+    [SerializeField]
+    private SlitPlate slitPlate;
+
+    private float waveLength;
+    private float distanceBetweenSlits;
+    private float distanceBetweenPlates;
+
+    private MeshRenderer meshRenderer;
+
+    void Start()
+    {
+        slitPlate = GameObject.Find("Plate").GetComponent<SlitPlate>();
```

Comment:
This line is not necessary. You are assigning the slitPlate already in the Unity Editor.